src/main/java/com/google/sps/data/UserOU.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.data;
+
+/**
+ The class represents an OU, with field name, path, parentPath, and depth. 
+ depth is the layer from root OU + 1.
+*/
+public class UserOU{
```

Comment:
This is fine, but in the future consider an AutoValue class, https://github.com/google/auto/blob/master/value/userguide/index.md

It helps with creating classes representing something and handles making all the getters/setters for you.